tests/async_http_client_test.py

```diff
@@ -115,7 +45,7 @@ def test_success_AsyncHTTP_response(self):
         ) as mock_Async:
             req = {
                 'method': 'GET',
-                'url': 'foo',
+                'url': 'http://foo',
```

Comment:
I think I'd prefer leaving it as is, as it is the convention used by python 'requests'.